spec.md

```diff
@@ -841,11 +841,46 @@ For success responses, the following fields are defined:
 | operation | string | For asynchronous responses, Service Brokers MAY return an identifier representing the operation. The value of this field MUST be provided by the Platform with requests to the [Last Operation](#polling-last-operation) endpoint in a percent-encoded query parameter. If present, MUST be a non-empty string. |
+| extension_apis | array-of-objects | For extensions to the Service Broker API, Service Brokers MAY return one or more `extension_api` objects that describe additional API endpoints via an OpenAPI document. See [Extension API Object](#extension-api-object) for more information. |
+
+##### Extension API Object
+
+The `extension_api` object MAY be used to describe any additional endpoint
+to the Open Service Broker API. An example of this could be lifecycle
+management of a Service Instance, (e.g. "Day Two Operations"), like Backup,
+Restore, Stop, Start, Restart and Pause.
+
+The `extension_api` MUST include a URI to an OpenAPI 3.0+ document that the
+Platform can use to determine the new endpoint(s), parameter(s) and
+authentication mechanism. The new APIs are extensions to the Open Service Broker
+API. As such they are indended to be invokved by the Platform on behalf of its
+clients.
+
+See [OpenAPI Server Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#serverObject) for more information.
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| discovery_url* | string | A URI pointing to a valid OpenAPI 3.0+ document describing the API extension(s) to the Open Service Broker API including server location, endpoints, parameters and any other detail the platform needs for invocation. The location of the API extension endpoint(s) can be local to the Service Broker or on a remote server. MUST be a valid URI. If `discovery_url` is a path, the Platform can assume it is to be called from the basepath of the Service Broker. The returned OpenAPI document MUST be in json format. See the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) for more information. |
+| server_url | string | A URI pointing to a remote server where API extensions will run. This URI will be used as the basepath for the endpoint(s) described by the `discovery_url` OpenAPI document. If no `server_url` is present, the Platform MUST assume the extension API endpoint(s) are to be invoked using the Service Broker host and port. If present, MUST be a valid URI. If `server_url` is a path, the Platform can assume it is to be called from the basepath of the Service Broker. |
```

Comment:
s/from the basepath/relative to the basepath/

Reply:
The text here says that if server_url is absent then the broker's host/port MUST be used, but if the OAPI doc has full URLs is that really true? Or can the OAPI doc only have relative paths?

Reply:
This approach is stating that we are ignoring any server object from the OAPI doc, and only paying attention to the path objects, (which are relative). If you have 'server_url' defined, you do server_url/+path_object1-x. If you don't have 'server_url' defined, you do servicebrokerhost:port/+path_object1-x.

Reply:
Programmatically I see how that works. From an OAPI perspective, is it normal to ignore the server object? Would we need to say something here about how Platforms will ignore server objects? Or should they reject OAPI docs that have server objects?

Reply:
In my experience, it's common practice to ignore the server object. You can also have more than one server object, so how do we decide which one to use? See, https://github.com/openservicebrokerapi/servicebroker/blob/master/openapi.yaml, we have two there and they are both basically examples. You typically always have a 'localhost' for testing during local development. I don't think there's a reason to explain this.

Reply:
I can't tell you how much that hurts my head  :-)

Reply:
mission accomplished.